src/resources/postcss/base/typography/_headings.pcss

```diff
@@ -75,7 +75,11 @@
 
 	.tribe-common-h6 {
 		@mixin heading;
-		@mixin heading-6;
+		@mixin mobile-heading-6;
+
+		@media (--viewport-medium) {
+			@mixin desktop-heading-6;
```

Comment:
this is good, there is also a `h6--min-medium` below that should use `desktop-heading-6` mixin below as well.